openpype/hosts/nuke/api/lib.py

```diff
@@ -1048,12 +1048,16 @@ def add_review_knob(node):
 def add_deadline_tab(node):
     node.addKnob(nuke.Tab_Knob("Deadline"))
 
+    knob = nuke.Int_Knob("deadlinePriority", "Priority")
+    knob.setValue(50)
+    node.addKnob(knob)
+
     knob = nuke.Int_Knob("deadlineChunkSize", "Chunk Size")
     knob.setValue(0)
     node.addKnob(knob)
 
-    knob = nuke.Int_Knob("deadlinePriority", "Priority")
-    knob.setValue(50)
+    knob = nuke.Int_Knob("deadlineConcurrentTasks", "Concurrent tasks")
+    knob.setValue(0)
```

Comment:
I feel like there should some documentation/hint somewhere that a value of zero means to take the default value from OpenPype settings.

Reply:
good point